languagetool-language-modules/nl/src/main/resources/org/languagetool/rules/nl/grammar.xml

```diff
@@ -384,6 +384,19 @@ regexp case-insensitive maken: (?i)
             <message>\3 is meervoud, dus het zou moeten zijn '<suggestion>zijn</suggestion> jouw \3'.</message>
             <example correction="zijn">Dit <marker>is</marker> jouw auto's.</example>
         </rule>
+        <rule id="JOU_JOUW_VNW" name="Waarom stuurt hij jouw (jou) geen berichtjes meer?" type="misspelling" default="temp_off">
+            <pattern>
+                <marker>
+                    <token>jouw</token>
+                </marker>
+                <token postag_regexp="yes" postag="VNW:OBP.*">
+                    <exception postag_regexp="yes" postag="VNW:OBP:ONV|VNW:OBP:VRB"/>
+                </token>
+            </pattern>
+            <message>U bedoelt waarschijnlijk '<suggestion>jou</suggestion>' of '<suggestion>je</suggestion>'.</message>
```

Comment:
Putting quotes directly around a `<suggestion>` element will cause the mvn test to fail (build break).
Removing these quotes should fix it.